aten/src/ATen/native/cuda/RowwiseScaledMM.cu

```diff
@@ -239,6 +239,7 @@ void f8f8bf16_rowwise_impl(
   StrideOutput stride_output = cutlass::make_cute_packed_stride(
       StrideOutput{}, cute::make_shape(M, static_cast<int>(out.stride(0)), 1));
 
+C10_DIAGNOSTIC_PUSH_AND_IGNORED_IF_DEFINED("-Wmissing-field-initializers")
```

Comment:
Are we sure this isn't our bug and we need to add some filed initializers to these kernels?

Reply:
I'm not able to judge because not familiar with cutlass.